cfgov/unprocessed/js/routes/common.js

```diff
@@ -26,3 +26,4 @@ require( '../modules/youtube' ).init();
 require( '../modules/pagination-validation.js' ).init();
 require( '../modules/show-hide-fields.js' ).init();
 require( '../modules/external-site-redirect.js' ).init();
+require( '../modules/multi-select.js' ).init();
```

Comment:
Eventually this'll make sense to require in the form organism instead of globally.